frontend/src/components/dialog/SettingsForm.vue

```diff
@@ -0,0 +1,171 @@
+<template>
+  <v-bottom-sheet
```

Comment:
does this move the sheet on the bottom?
On my 4k monitor i had to search a little before i found the form.

![grafik](https://github.com/ecamp/ecamp3/assets/1506818/40228dab-84d0-4683-aa84-9e471019c5da)

Reply:
Changed design to dialog